module/config.mjs

```diff
@@ -113,6 +113,12 @@ ED4E.globalBonuses = {
 	allRangedDamage: {
 		label: 			"ED.Actor.GlobalBonus.allRangedDamage"
 	},
+	allRecoveryEffects: {
```

Comment:
Can we try to introduce some code internal distinction between recovery test the process/test vs. recovery test the ressource?

With this name this would in my mind then include things that use the ressource?

Reply:
unsure, this effect will add x step to every recovery you do. we can certainly add "resource" to all Resources we have

Reply:
Rename resourcen into recoveryResource.